app_dart/lib/src/request_handlers/append_log.dart

```diff
@@ -34,11 +37,14 @@ class AppendLog extends ApiRequestHandler<Body> {
             requestBodyValue: requestBodyValue);
 
   final StackdriverLoggerService stackdriverLogger;
+  final DatastoreServiceProvider datastoreProvider;
 
   static const String ownerKeyParam = 'ownerKey';
 
   @override
   Future<Body> post() async {
+    final DatastoreService datastore = datastoreProvider(
+        db: config.db, maxEntityGroups: config.maxEntityGroups);
```

Comment:
With the refactor, I don't think `maxEntityGroups` needs to live in the `CocoonConfig`. It could be moved into `DatastoreService` and the `maxEntityGroups` parameter could be removed.

Reply:
I still need to update a couple of handlers. I'll move the maxEntityGroups to datastore once all the handlers are using it and I'll remove it from CocoonConfig.